testsuite/MDAnalysisTests/core/test_atomselections.py

```diff
@@ -209,6 +209,8 @@ def test_sphzone(self, universe, selstr):
     def test_cylayer(self, universe, selstr):
         sel = universe.select_atoms(selstr)
         assert_equal(len(sel), 88)
+        empty = universe.select_atoms('cylayer 4.0 6.0 10 -10 name NOT_A_NAME')
```

Comment:
Thanks, these are thorough tests -- could you make each of these separate though? That way each function tests one specific functionality.

Reply:
Yep! just fixed this!